eng/pipelines/mono/templates/build-job.yml

```diff
@@ -0,0 +1,88 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osSubgroup: ''
+  platform: ''
+  container: ''
+  timeoutInMinutes: ''
+  stagedBuild: false
+  variables: {}
+  pool: ''
+  condition: true
+
+### Product build
+jobs:
+- template: xplat-pipeline-job.yml
+  parameters:
+    buildConfig: ${{ parameters.buildConfig }}
+    _BuildConfig: ${{ parameters.buildConfig }}
```

Comment:
This parameter is not used in mono/xplat-pipeline-job.yml, not even defined as a parameter in the template.